lib/cli/package.json

```diff
@@ -14,17 +14,21 @@
   "license": "MIT",
   "author": "Storybook Team",
   "bin": {
-    "getstorybook": "./bin/generate.js"
+    "getstorybook": "./bin/index.js"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/storybooks/storybook.git"
   },
   "scripts": {
+    "pretest": "cd test && ./update_fixtures.sh",
```

Comment:
Can we set this up so this is a jest project, like how the examples/react-native-vanilla is set up?

This would allow running the tests along with all the other unit tests!

Reply:
I'll take a look

Reply:
> all the other unit tests!

These tests are not unit =)